cf-agent/verify_files_utils.c

```diff
@@ -3596,13 +3596,13 @@ bool VerifyOwner(EvalContext *ctx, const char *file, const Promise *pp, Attribut
 
     if (attr.perms.groups->next == NULL && attr.perms.groups->gid == CF_UNKNOWN_GROUP)  // Only one non.existent item
     {
-        cfPS(ctx, LOG_LEVEL_ERR, PROMISE_RESULT_FAIL, pp, attr, "Unable to make file belong to an unknown group");
+        cfPS(ctx, LOG_LEVEL_ERR, PROMISE_RESULT_FAIL, pp, attr, "Unable to make file belong to an unknown group '%s'", attr.perms.groups->gid );
         *result = PromiseResultUpdate(*result, PROMISE_RESULT_FAIL);
```

Comment:
We don't need to print the GID/UID here, we know that it is `CF_UNKNOWN_GROUP` or `CF_UNKNOWN_OWNER`. From `cf3.defs.h`:

```c
#define CF_SAME_OWNER ((uid_t)-1)
#define CF_UNKNOWN_OWNER ((uid_t)-2)
#define CF_SAME_GROUP ((gid_t)-1)
#define CF_UNKNOWN_GROUP ((gid_t)-2)
```